extensions/ohmyzsh-git-alias/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "oh-my-zsh-git-alias",
+  "title": "Oh My Zsh Git Alias",
+  "description": "Quick search git alias.",
+  "icon": "icon.png",
+  "author": "yufeikang",
+  "categories": [
+    "Developer Tools",
+    "Documentation"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Git Alias",
+      "description": "Search Oh-My-Zsh Git Alias",
+      "mode": "view"
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.45.2",
+    "axios": "^1.2.1"
```

Comment:
This doesnt look like it's being used, can we remove and ensure pacakge-lock is updated. Thanks!